bindings/nodejs/src/lib.rs

```diff
@@ -311,6 +324,45 @@ impl Lister {
     }
 }
 
+#[napi(iterator)]
+pub struct BlockingLister(opendal::BlockingLister);
+
+#[napi]
+#[allow(clippy::should_implement_trait)] /*
+                                         Method `next` can be confused for the standard trait method `std::iter::Iterator::next`.
+                                         But in JavaScript, it is also customary to use the next method directly to obtain the next element.
+                                         Therefore, disable this clippy. It can be removed after a complete implementation of Generator.
+                                         FYI: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator
+                                         */
+impl BlockingLister {
+    #[napi]
+    pub fn next(&mut self) -> Result<Option<Entry>> {
```

Comment:
Please don't export next as public api.

Reply:
https://github.com/datafuselabs/opendal/blob/a540559515e6e006430d4ff93ad866cd17f4dd24/bindings/nodejs/__test__/index.spec.ts#L73-L78

I am providing usage methods similar to the above code, as we currently cannot fully implement System.iterator.

Reply:
> I am providing usage methods similar to the above code, as we currently cannot fully implement Symbol.iterator.

I prefer to offer only one way of using our public API. If `Symbol.iterator` is not yet available, let's remove it and stick with just `next()`.